static/scripts/main.js

```diff
@@ -6,23 +6,39 @@ window.onload = function () {
     if(hash) {
       hash.scrollIntoView()
     }
+    $('[data-md-sticky-header]').headroom({
+      offset: UnpinOffset,
+      tolerance: 0
+    });
 }
 
-var CurrentScroll = 0;
+var SearchbarTop = 61;
+var UnpinOffset = 400;
```

Comment:
This variable is used when applying `headroom` on the header at line 9, however the variable is declared at line 15, which could cause some confusion. Can we move this variable to a place before it's used?